multiqc/modules/fgbio/groupreadsbyumi.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env python
+
+""" MultiQC module to parse output from fgbio GroupReadsByUmi
+"""
+
+############################################################
+######  LOOKING FOR AN EXAMPLE OF HOW MULTIQC WORKS?  ######
+############################################################
+#### Stop! This is one of the most complicated modules. ####
+#### Have a look at Kallisto for a simpler example.     ####
+############################################################
+
+from __future__ import print_function
+from collections import OrderedDict
+import io
+import json
+import logging
+import os
+import re
+import zipfile
+
+from multiqc import config
+from multiqc.plots import linegraph
+from multiqc.modules.base_module import BaseMultiqcModule
+
+# Initialise the logger
+log = logging.getLogger(__name__)
+
+class MultiqcModule(BaseMultiqcModule):
+
+    def __init__(self):
+
+        # Initialise the parent object
+        super(MultiqcModule, self).__init__(name='Fgbio', anchor='fgbio',
+        href="http://fulcrumgenomics.github.io/fgbio/tools/latest/GroupReadsByUmi.html",
+        info="fgbio is a command line toolkit for working with genomic and particularly next generation sequencing data.")
+
+        self.umi_data = dict()
+
+        # Find and parse unzipped FastQC reports
+        for f in self.find_log_files('fgbio/groupreadsbyumi'):
+            self.parse_hist_report(f)
+
```

Comment:
If no logs are found, we need to raise a `UserWarning` here to stop the module code going any further.

https://multiqc.info/docs/#no-files-found

Reply:
Please also filter the samples and print to the sources file:

https://multiqc.info/docs/#filtering-by-parsed-sample-names
https://multiqc.info/docs/#printing-to-the-sources-file

Reply:
Warning is now raised when no logs are found. Sample filtering should now work. printing-to-the-sources-files should also be working